src/com/owncloud/android/providers/UsersAndGroupsSearchProvider.java

```diff
@@ -70,8 +72,14 @@
 
     public static final String AUTHORITY = UsersAndGroupsSearchProvider.class.getCanonicalName();
     public static final String ACTION_SHARE_WITH = AUTHORITY + ".action.SHARE_WITH";
-    public static final String DATA_USER = AUTHORITY + ".data.user";
-    public static final String DATA_GROUP = AUTHORITY + ".data.group";
+
+    public static final String USER = "user";
+    public static final String GROUP = "group";
+    public static final String REMOTE = "remote";
+
+    public static final String DATA_USER = AUTHORITY + ".data." + USER;
+    public static final String DATA_GROUP = AUTHORITY + ".data." + GROUP;
+    public static final String DATA_REMOTE = AUTHORITY + ".data." + REMOTE;
```

Comment:
If we build the String this way then ".data." should also be declared as a constant :)